common/static/common/js/discussion/utils.js

```diff
@@ -194,18 +194,6 @@
             params.url = URI(params.url).addSearch({
                 ajax: 1
             });
-            params.beforeSend = function() {
```

Comment:
@bradenmacdonald added in https://github.com/edx/edx-platform/pull/12914/commits/24b27fab925c2f93512aaa6d92606e0a5a494ace